templates/local.conf.j2

```diff
@@ -0,0 +1,16 @@
+[Time]
```

Comment:
For a drop-in file like this, please add as the very first line:

```
# {{ ansible_managed }}
```

**Note:** I didn't do this with the previous template, because I'm an advocate of no diff if no variables are set.